client/app/components/LoadingIndicator/tests/__snapshots__/index.test.js.snap

```diff
@@ -0,0 +1,379 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<LoadingIndicator /> should match the snapshot 1`] = `
+.c0 {
+  margin: 2em auto;
+  width: 40px;
+  height: 40px;
+  position: relative;
+}
+
+.c1 {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  top: 0;
+}
+
+.c1:before {
+  content: '';
+  display: block;
+  margin: 0 auto;
+  width: 15%;
+  height: 15%;
+  background-color: #999;
+  border-radius: 100%;
+  -webkit-animation: evVlQQ 1.2s infinite ease-in-out both;
+  animation: evVlQQ 1.2s infinite ease-in-out both;
+}
+
+.c2 {
+  width: 100%;
+  height: 100%;
+  position: absolute;
+  left: 0;
+  top: 0;
+  -webkit-transform: rotate(30deg);
+  -ms-transform: rotate(30deg);
+  -webkit-transform: rotate(30deg);
+  -ms-transform: rotate(30deg);
+  transform: rotate(30deg);
```

Comment:
You should export this method from a helper.js file, like that:
export const rotate = (rotateValue = '0deg') => `
  -webkit-transform: rotate(rotateValue);
  -ms-transform: rotate(rotateValue);
  -webkit-transform: rotate(rotateValue);
  -ms-transform: rotate(rotateValue);
  transform: rotate(rotateValue);
`
then use it when needed:
.c2 {
  left: 0;
  top: 0;
  ${rotate(30deg)};
}

or if you need to check some conditions before use it:
${({ needToRotate }) => rotate(30deg)}